CONTRIBUTING.rst

```diff
@@ -0,0 +1,100 @@
+- Fix typos
+- Write documentation
+- Contribute your plugins
+- Improve the Brigade core
+
+Spread the word about Brigade
+-----------------------------
+
+Even if you aren't in the position that you can contribute your time to this project, it still helps us if you spread the word about the project. It could be just a short notice in social media or a discussion you have with your friends. As more people become aware of the project there's a better chance that we reach people who are able to contribute. So, even if you can't directly contribute yourself, someone you refer to us might.
+
+Suggesting new features
+-----------------------
+
+It could be that you are aware of something that would be great to have in Brigade and we are always welcoming feature requests. Make sure you explain in what scenario your suggested feature would be useful.
+
+Reporting bugs
+--------------
+
+When you are `reporting bugs <https://github.com/brigade-automation/brigade/issues>`_, make sure that you give a explaination about the outcome that you expect and what you are seeing. The bugs which are hardest to fix are the ones which we are unable to reproduce. For this reason it's important that you describe what you did and show us how we can reproduce the bug in another environment.
+
+Fix typos
+---------
+
+While we try to take care, getting all the works correct can be.. differcult. Typos are the easiest things to fix and if you find any you can help us from looking silly. You can find more typos to fix by looking in the `Brigade source code <https://github.com/brigade-automation/brigade/tree/develop/brigade>`_ or by visiting the `Brigade documentation <https://brigade.readthedocs.io>`_.
+
+Writing documentation
+---------------------
+
+Documentation is another great way to help if you don't want to contribute actual code. The documentation of Brigade is divided into different sections.
+
+- Tutorials: Aims to help people learn Brigade with a lot of handholding, the user might not end up with something useful after following the tutorial. The goal is for people to learn how to use Brigade.
+- How-to guides: This sections goal is to help people solve a specific task with Brigade
+- Reference guides: This section describe the Brigade API and plugins. Most of the content in this area is generated from the source code itself.
+
+Contributions to the documentation can be small fixes such as changing scentences to make the text more clear, or it could be new guides.
+
+Contributing plugins
+--------------------
+
+If you have written your custom plugin for Brigade there's a good chance that it can be useful for others as well. General guidelines when writing plugins are:
+
+- Make them as generic as possible, it doesn't help others if they only work in your environment
+- Make sure that it's possible to have unit tests which automatically test that the plugins are working
+
+
+Contributing to the Brigade core
+--------------------------------
+
+[xyz]
```

Comment:
I would just add here to make sure that existing tests pass and to add tests for the new code to make sure future people working around that code doesn't break anything.